src/core/client/admin/components/BanModal.tsx

```diff
@@ -218,7 +218,7 @@ const BanModal: FunctionComponent<Props> = ({
           message: customizeMessage ? emailMessage : getDefaultMessage,
           rejectExistingComments,
           siteIDs: viewerIsScoped
-            ? viewer.moderationScopes?.sites!.map(({ id }) => id)
+            ? viewer?.moderationScopes?.sites?.map(({ id }) => id)
```

Comment:
The `!` operator in TS is such a footgun, nice clean up!

Reply:
🙏